packages/twenty-front/src/modules/views/components/EditableFilterDropdownButton.tsx

```diff
@@ -57,7 +68,15 @@ export const EditableFilterDropdownButton = ({
           />
         }
         dropdownComponents={
-          <ObjectFilterDropdownFilterInput filterDropdownId={recordFilter.id} />
+          isSearchFilter ? (
+            <ViewBarFilterDropdownSearchInput
```

Comment:
Could you put that inside `ObjectFilterDropdownFilterInput` which is already doing a switch of all possible cases ?